torchsynth/synth.py

```diff
@@ -243,24 +244,35 @@ def save_hyperparameters(self, filename: str) -> None:
         """
         Save hyperparameters to a JSON file
         """
+        # Render all hyperparameters as JSON
         hp = [{"name": key, "value": val} for key, val in self.hyperparameters.items()]
+        hp = json.dumps(hp, indent=True)
+        hp = util.format_hyperparameter_json(hp)
+
         with open(os.path.abspath(filename), "w") as fp:
-            json.dump(hp, fp, indent=True)
+            fp.write(hp)
 
-    def load_hyperparameters(self, name: str) -> None:
+    def load_hyperparameters(self, nebula: str) -> None:
```

Comment:
Decided to keep this as is. If we add multiple args here for the name vs. filename then we would need to have that in the Voice constructor as well. Unless we decide that we only support loading of torchsynth package nebulae from a synth constructor.

Also. Do we want to call these functions `load_nebula` and `save_nebula`?

Reply:
Why not? Do it in the next PR I guess :)

Maybe that means we also have to change hyperparameters in synth to nebula?

Reply:
Will leave for a future PR -- made an issue for it #330